lib/cc/cli/help.rb

```diff
@@ -14,7 +14,7 @@ def run
 
       def commands
         [
-          "analyze [-f format] [-e engine] <path>",
+          "analyze [-f format] [-e engine(:channel)] <path>",
```

Comment:
This is awesome - we should highlight this when documenting upgrading to the new eslint